src/app/home/home.component.html

```diff
@@ -1,93 +1,67 @@
-<div id="home" class="container content">
-    <div class="row">
-        <div class="col-md-4">
-        </div>
-        <div class="col-md-2">
-            <a class="btn btn-default btn-block" type="button" href="https://fabric8.io/guide/getStarted/index.html">Learn</a>
-        </div>
-        <div class="col-md-2">
-            <button class="btn btn-default btn-block" type="button" (click)="createSpace.open()">
-                Start Collaborating
-            </button>
-        </div>
-        <div class="col-md-4"></div>
+<div class="container-fluid container-cards-pf">
+  <div class="row">
+    <div class="col-xs-12">
+          <div class="btn-group">
+                <a class="btn btn-default " type="button" href="https://fabric8.io/guide/getStarted/index.html" >Learn</a>
+                <button class="btn btn-default " type="button" (click)="createSpace.open()" >Start Collaborating</button>
+          </div>
     </div>
-    <div class="row">
-        <div class="col-md-4">
-            <h2>Recent Collaboration Spaces</h2>
-            <div>
-                <a (click)="createSpace.open()">New</a> | <a [routerLink]="['/pmuir/spaces']">Browse</a>
-            </div>
-            <div class="container-fluid container-cards-pf">
-                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
-                    <div class="card-pf card-pf-view ">
-                        <div class="card-pf-body">
-                            <div class="card-pf-top-element">
-                            </div>
-                            <h2 class="card-pf-title text-center">
-                            </h2>
-                            <div class="card-pf-items">
-                                <ul class="list-unstyled">
-                                    <li *ngFor="let s of dummy.spaces"><a [routerLink]="[ s.path]">{{s.name}}</a></li>
-                                </ul>
-                            </div>
-                        </div>
-                        <div class="card-pf-view-checkbox">
-                            <input type="checkbox">
-                        </div>
-                    </div>
-                </div>
-            </div>
+  </div>
+  <div class="row row-cards-pf">
+    <div class="col-sm-4">
+      <div class="card-pf card-pf-accented">
+        <div class="card-pf-title">
+          <h4>Recent Collaboration Spaces</h4>
+          <a (click)="createSpace.open()">New</a> | <a [routerLink]="['/pmuir/spaces']">Browse</a>
```

Comment:
yep .. not a layout problem... content shows up when refresh browser ...pointing to another rendering issue ...